.travis.yml

```diff
@@ -1,7 +1,7 @@
 ---
 language: node_js
 node_js:
-  - "10"
+  - '10'
```

Comment:
Please revert this back to the original. It is not required to be changed

Reply:
ok